lib-find-meal/src/main/res/layout/find_meal_fragment.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".view.FindMealFragment">
+
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
```

Comment:
Nice! I've never used one of these so TIL

Reply:
The SwipeRefreshLayout? @dsmith42 did that :)

Reply:
We yanked it from the kmm tutorial 😃